### ISSUE TITLE
Does not work on second monitor

### ISSUE THREAD

**Issue by jojonaloha** [nvkelso/color-oracle-java#6]
I have a second monitor plugged in and the app does not appear to work on the second monitor. I'm using a Macbook Pro with a Thunderbolt display and the Thunderbolt does not get the same treatment when I switch between Normal Vision and the other options.

I was able to get it to work if I changed the Thunderbolt to my primary display or by Mirroring the display.

**Comment by nvkelso** [comment]
I believe this is a known limitation of the software, no plans to fix at this time (but Pull Requests are welcome).

**Comment by JiaChen-Zeng** [comment]
This problem exists.

**Comment by m21-cerutti** [comment]
Yeah, it's a bit anoying if we work with multiple monitors, it repeat the main monitor on all screens.

![2022_08_19_10-31-57](https://user-images.githubusercontent.com/23432929/185579132-bdfd899b-9068-4f77-b5f2-9c08ba4c0cf9.png)

Why not just aply it on the main monitor or better, the monitor we choose in parameters ?
